.travis.yml

```diff
@@ -35,7 +35,7 @@ before_script: 'if [[ "$MAIN_STAGE" = true ]]; then greenkeeper-lockfile-update;
 script:
   - npm run build
   - 'if [[ "$MAIN_STAGE" = true ]]; then ./scripts/check-pristine-state package-lock.json; fi'
-  - 'if [[ "$MAIN_STAGE" = true ]]; then Example/Pods/SwiftLint/swiftlint lint --strict --config Example/.swiftlint.yml; fi'
+  - 'if [[ "$MAIN_STAGE" = true ]]; then (cd Example && Pods/SwiftLint/swiftlint lint --strict --config .swiftlint.yml); fi'
```

Comment:
What code is being excluded at the moment?

If I try your suggestion, then SwiftLint tries to run on Pods when running locally:
<img width="868" alt="Screen Shot 2019-08-12 at 17 53 59" src="https://user-images.githubusercontent.com/3900360/62878870-40814300-bd2a-11e9-913d-ce0f1d5bc307.png">
![Screen Shot 2019-08-12 at 17 52 09](https://user-images.githubusercontent.com/3900360/62878734-f39d6c80-bd29-11e9-8096-c0a770d3852f.png)

On the other hand, if I keep my changes, I see 51 files being linted locally and also the same 52 files being linted on the CI. I wonder what code you see being excluded?

Reply:
Well any swift code in `./Backpack` isn't being linted, (which at the moment is only your code)

Reply:
I see those being linted on the CI:
<img width="1018" alt="Screen Shot 2019-08-12 at 18 05 47" src="https://user-images.githubusercontent.com/3900360/62879630-d6699d80-bd2b-11e9-8710-80eb894dc733.png">

And also locally:
![Screen Shot 2019-08-12 at 18 06 30](https://user-images.githubusercontent.com/3900360/62879667-ef724e80-bd2b-11e9-84e5-63c4638e486f.png)

Reply:
That seems strange because if we cd into a different directory then it's not gonna have any knowledge of those files.

When I do this locally I can see swift files in Backpack being excluded due to the `cd` command:
```
 georgegillams@GEORGILL01M  ~/Documents/backpack-ios   master 13.0.0 ?  Example/Pods/SwiftLint/swiftlint lint --strict --config Example/.swiftlint.yml                                                                                                                                              ✔  10017  13:40:31
Loading configuration from 'Example/.swiftlint.yml'
Linting Swift files at paths
Linting 'ThemeRegistry.swift' (1/46)
Linting 'Settings.swift' (2/46)
Linting 'IconSwiftTest.swift' (3/46)
Linting 'TextViewUITest.swift' (4/46)
Linting 'TextFieldUITest.swift' (5/46)
Linting 'HorizontalNavigationUITest.swift' (6/46)
Linting 'ThemeHelpers.swift' (7/46)
Linting 'ChipUITest.swift' (8/46)
Linting 'DialogUITest.swift' (9/46)
Linting 'ExampleApp.swift' (10/46)
Linting 'TextViewsViewController.swift' (11/46)
Linting 'DialogSelectorViewController.swift' (12/46)
Linting 'CardsSelectorViewController.swift' (13/46)
Linting 'HorizontalNavViewController.swift' (14/46)
Linting 'ProgressBarViewController.swift' (15/46)
Linting 'LabelsSelectorTableViewController.swift' (16/46)
Linting 'TappableLinkLabelsViewController.swift' (17/46)
Linting 'GradientViewController.swift' (18/46)
Linting 'ChipsViewController.swift' (19/46)
Linting 'StarRatingsViewController.swift' (20/46)
Linting 'CardsViewController.swift' (21/46)
Linting 'SpinnersViewController.swift' (22/46)
Linting 'DialogViewController.swift' (23/46)
Linting 'HorizontalNavSelectorViewController.swift' (24/46)
Linting 'GradientSelectorTableViewController.swift' (25/46)
Linting 'ColorsViewController.swift' (26/46)
Linting 'SwitchesViewController.swift' (27/46)
Linting 'DividedCardsViewController.swift' (28/46)
Linting 'LabelsPerformanceViewController.swift' (29/46)
Linting 'LabelsViewController.swift' (30/46)
Linting 'TappableLinkLabelsSelectorViewController.swift' (31/46)
Linting 'TextFieldsViewController.swift' (32/46)
Linting 'CalendarViewController.swift' (33/46)
Linting 'BadgesViewController.swift' (34/46)
Linting 'LabelMultiFontStyleViewController.swift' (35/46)
Linting 'IconsViewController.swift' (36/46)
Linting 'SettingsViewController.swift' (37/46)
Linting 'ColorPreviewCollectionViewCell.swift' (38/46)
Linting 'ChipPreviewCollectionViewCell.swift' (39/46)
Linting 'IconsPreviewCollectionViewCell.swift' (40/46)
Linting 'UICollectionViewMasonryFlowLayout.swift' (41/46)
Linting 'PreviewCollectionViewHeader.swift' (42/46)
Linting 'BPKBadgeContainer.swift' (43/46)
Linting 'BPKTableViewSelectableCell.swift' (44/46)
Linting 'UIWindow+Utility.swift' (45/46)
Linting 'file-in-Backpack.swift' (46/46)
/Users/georgegillams/Documents/backpack-ios/Backpack/Badge/file-in-Backpack.swift:1:1: warning: Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)
Done linting! Found 1 violation, 0 serious in 46 files.
 georgegillams@GEORGILL01M  ~/Documents/backpack-ios   master 13.0.0 ?  (cd Example/ && Pods/SwiftLint/swiftlint lint --strict --config .swiftlint.yml)                                                                                                                                           3 ↵  10018  13:40:35
Reverting to nvm default version
Now using node v10.15.3 (npm v6.9.0)
Loading configuration from '.swiftlint.yml'
Linting Swift files at paths
Linting 'IconSwiftTest.swift' (1/45)
Linting 'Settings.swift' (2/45)
Linting 'ThemeRegistry.swift' (3/45)
Linting 'TextViewUITest.swift' (4/45)
Linting 'HorizontalNavigationUITest.swift' (5/45)
Linting 'TextFieldUITest.swift' (6/45)
Linting 'ThemeHelpers.swift' (7/45)
Linting 'ChipUITest.swift' (8/45)
Linting 'DialogUITest.swift' (9/45)
Linting 'ExampleApp.swift' (10/45)
Linting 'TextViewsViewController.swift' (11/45)
Linting 'DialogSelectorViewController.swift' (12/45)
Linting 'CardsSelectorViewController.swift' (13/45)
Linting 'HorizontalNavViewController.swift' (14/45)
Linting 'ProgressBarViewController.swift' (15/45)
Linting 'LabelsSelectorTableViewController.swift' (16/45)
Linting 'TappableLinkLabelsViewController.swift' (17/45)
Linting 'GradientViewController.swift' (18/45)
Linting 'ChipsViewController.swift' (19/45)
Linting 'StarRatingsViewController.swift' (20/45)
Linting 'CardsViewController.swift' (21/45)
Linting 'SpinnersViewController.swift' (22/45)
Linting 'DialogViewController.swift' (23/45)
Linting 'HorizontalNavSelectorViewController.swift' (24/45)
Linting 'GradientSelectorTableViewController.swift' (25/45)
Linting 'ColorsViewController.swift' (26/45)
Linting 'SwitchesViewController.swift' (27/45)
Linting 'DividedCardsViewController.swift' (28/45)
Linting 'LabelsPerformanceViewController.swift' (29/45)
Linting 'LabelsViewController.swift' (30/45)
Linting 'TappableLinkLabelsSelectorViewController.swift' (31/45)
Linting 'TextFieldsViewController.swift' (32/45)
Linting 'CalendarViewController.swift' (33/45)
Linting 'BadgesViewController.swift' (34/45)
Linting 'LabelMultiFontStyleViewController.swift' (35/45)
Linting 'IconsViewController.swift' (36/45)
Linting 'SettingsViewController.swift' (37/45)
Linting 'ColorPreviewCollectionViewCell.swift' (38/45)
Linting 'ChipPreviewCollectionViewCell.swift' (39/45)
Linting 'IconsPreviewCollectionViewCell.swift' (40/45)
Linting 'UICollectionViewMasonryFlowLayout.swift' (41/45)
Linting 'PreviewCollectionViewHeader.swift' (42/45)
Linting 'BPKBadgeContainer.swift' (43/45)
Linting 'BPKTableViewSelectableCell.swift' (44/45)
Linting 'UIWindow+Utility.swift' (45/45)
Done linting! Found 0 violations, 0 serious in 45 files.
```

Reply:
See how `file-in-Backpack.swift` only gets picked up on the first run?

Reply:
Second run is working fine for me:
```bash
FEDETRIM01M:backpack-ios federicotrimboli$ (cd Example/ && Pods/SwiftLint/swiftlint lint --strict --config .swiftlint.yml)
Loading configuration from '.swiftlint.yml'
Linting Swift files at paths 
Linting 'BottomSheet.swift' (1/51)
Linting 'BackpackFloatingPanelController.swift' (2/51)
Linting 'IconSwiftTest.swift' (3/51)
Linting 'Storyboards.swift' (4/51)
Linting 'ThemeRegistry.swift' (5/51)
Linting 'Settings.swift' (6/51)
Linting 'TextViewUITest.swift' (7/51)
Linting 'ThemeHelpers.swift' (8/51)
Linting 'HorizontalNavigationUITest.swift' (9/51)
Linting 'ChipUITest.swift' (10/51)
Linting 'TextFieldUITest.swift' (11/51)
Linting 'DialogUITest.swift' (12/51)
Linting 'ExampleApp.swift' (13/51)
Linting 'TextViewsViewController.swift' (14/51)
Linting 'DialogSelectorViewController.swift' (15/51)
Linting 'CardsSelectorViewController.swift' (16/51)
Linting 'HorizontalNavViewController.swift' (17/51)
Linting 'ProgressBarViewController.swift' (18/51)
Linting 'LabelsSelectorTableViewController.swift' (19/51)
Linting 'TappableLinkLabelsViewController.swift' (20/51)
Linting 'GradientViewController.swift' (21/51)
Linting 'ChipsViewController.swift' (22/51)
Linting 'StarRatingsViewController.swift' (23/51)
Linting 'CardsViewController.swift' (24/51)
Linting 'SpinnersViewController.swift' (25/51)
Linting 'DialogViewController.swift' (26/51)
Linting 'HorizontalNavSelectorViewController.swift' (27/51)
Linting 'GradientSelectorTableViewController.swift' (28/51)
Linting 'BottomSheetContentViewController.swift' (29/51)
Linting 'BottomSheetViewController.swift' (30/51)
Linting 'BottomSheetBottomSectionViewController.swift' (31/51)
Linting 'ColorsViewController.swift' (32/51)
Linting 'SwitchesViewController.swift' (33/51)
Linting 'DividedCardsViewController.swift' (34/51)
Linting 'LabelsPerformanceViewController.swift' (35/51)
Linting 'LabelsViewController.swift' (36/51)
Linting 'TappableLinkLabelsSelectorViewController.swift' (37/51)
Linting 'TextFieldsViewController.swift' (38/51)
Linting 'CalendarViewController.swift' (39/51)
Linting 'BadgesViewController.swift' (40/51)
Linting 'LabelMultiFontStyleViewController.swift' (41/51)
Linting 'IconsViewController.swift' (42/51)
Linting 'SettingsViewController.swift' (43/51)
Linting 'ColorPreviewCollectionViewCell.swift' (44/51)
Linting 'ChipPreviewCollectionViewCell.swift' (45/51)
Linting 'IconsPreviewCollectionViewCell.swift' (46/51)
Linting 'UICollectionViewMasonryFlowLayout.swift' (47/51)
Linting 'PreviewCollectionViewHeader.swift' (48/51)
Linting 'BPKBadgeContainer.swift' (49/51)
Linting 'BPKTableViewSelectableCell.swift' (50/51)
Linting 'UIWindow+Utility.swift' (51/51)
Done linting! Found 0 violations, 0 serious in 51 files.
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...9 completed.

```

Reply:
Looks 💯 to me 👍